### PR TITLE
Update configure-unit-tests-by-using-a-dot-runsettings-file.md

### PR DESCRIPTION
**Pull request by BBreiden**
Add documentation for TreatDiscoveryWarningsAsErrors setting as suggested in this MSTest issue https://github.com/microsoft/testfx/issues/1736

### CODE REVIEW

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -329,6 +329,7 @@ These settings are specific to the test adapter that runs test methods that have
 |**MapNotRunnableToFailed**|true|A value indicating whether a not runnable result is mapped to failed test.|
 |**Parallelize**|Used to set the parallelization settings:<br /><br />**Workers**: The number of threads/workers to be used for parallelization, which is by default **the number of processors on the current machine**.<br /><br />**SCOPE**: The scope of parallelization. You can set it to **MethodLevel**. By default, it's **ClassLevel**.<br /><br />`<Parallelize><Workers>32</Workers><Scope>MethodLevel</Scope></Parallelize>`|
 |**TestTimeout**|0|Gets specified global test case timeout.|
+|**TreatDiscoveryWarningsAsErrors**|false|To report test discovery warnings as errors, set this valie to **true**.|
```
Comment (Evangelink) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:332]:
```suggestion
|**TreatDiscoveryWarningsAsErrors**|false|To report test discovery warnings as errors, set this value to **true**.|
```

**Review by Evangelink — APPROVED** [review]
LGTM - a small typo.

Thanks for your contribution @BBreiden <3

**Comment by Court72** [comment]
@Mikejo5000

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off Thanks!

**Pull request merged**